xchange-lakebtc/pom.xml

```diff
@@ -25,6 +25,12 @@
 	<dependencies>
 
 		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.3.2</version>
+		</dependency>
```

Comment:
Done, but I have difficulties uploading the jar to nexus, in case you can help with that, I would appreciate it.